components/x-gift-article/storybook/index.jsx

```diff
@@ -0,0 +1,108 @@
+const { GiftArticle } = require('../dist/GiftArticle.cjs')
+import fetchMock from 'fetch-mock'
+import React from 'react'
+import { Helmet } from 'react-helmet'
+import BuildService from '../../../.storybook/build-service'
+
+const dependencies = {
+	'o-fonts': '^3.0.0'
+}
+
+export default {
+	title: 'x-gift-article'
+}
+
+export const WithGiftCredits = (args) => {
+	require('./with-gift-credits').fetchMock(fetchMock)
+	return (
+		<div className="story-container">
+			{dependencies && <BuildService dependencies={dependencies} />}
+			<Helmet>
+				<link rel="stylesheet" href={`components/x-gift-article/dist/GiftArticle.css`} />
```

Comment:
Does `/dist/GiftArticle.css` get updated on changes?

Reply:
I think you would have to rebuild, since when I save something it looks like the live-reloader is only looking for `.jsx` files to update

Reply:
to get around this in the future we could potentially add `style-loader` to the storypack webpack config, but i think this is fine for now

Reply:
@antonsamper  We had the answer to yesterday's bug right here 😱!!